Sources/Build/Error.swift

```diff
@@ -23,6 +24,8 @@ extension Error: FixableError {
             return "no modules found"
         case .onlyCModule(let name):
             return "only system module package \(name) found"
+        case .incompatibleToolsVersions(let module):
```

Comment:
Yep but we don't have that data structure yet, it will come once we merge #926.
I'll add it then